Bottom: 3d1a7a453d81a0e630b788865b457094f5c876be
Top:    78d652b6b03f4394cc0479eb2d323e224bf42db0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 23:34:29 +0100

hid/gtk: Remove unused function ghid_netlist_nodes_update ()


---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 26b97bb..7ec5b95 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -111,9 +111,6 @@ static LibraryMenuType *node_selected_net;
    |		PCB calls this to tell the gui netlist code the layout net has
    |		changed and the gui data structures (net and optionally node data
    |		models) should be rebuilt.
-   |
-   |   ghid_netlist_nodes_update(LibraryMenuType *net)
-   |		Called when the node model should be updated to a netlist.
 */
 
 
@@ -965,12 +962,6 @@ ghid_netlist_highlight_node (gchar * node_name)
     while (gtk_tree_model_iter_next (node_model, &iter));
 }
 
-void
-ghid_netlist_nodes_update (LibraryMenuType * net)
-{
-  node_model_update (net);
-}
-
 /* If code in PCB should change the netlist, call this to update
    |  what's in the netlist window.
 */
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 930df19..114574f 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -441,7 +441,6 @@ void ghid_set_status_line_label (void);
 void ghid_netlist_window_create (GHidPort * out);
 void ghid_netlist_window_show (GHidPort * out, gboolean raise);
 void ghid_netlist_window_update (gboolean init_nodes);
-void ghid_netlist_nodes_update (LibraryMenuType * net);
 
 LibraryMenuType *ghid_get_net_from_node_name (gchar * name, gboolean);
 void ghid_netlist_highlight_node (gchar * name);
